src/cmd/services/m3coordinator/downsample/downsampler_test.go

```diff
@@ -1881,7 +1881,7 @@ func testDownsamplerAggregationIngest(
 
 		samplesAppender := samplesAppenderResult.SamplesAppender
 		for _, sample := range metric.samples {
-			err = samplesAppender.AppendCounterSample(sample)
+			err = samplesAppender.AppendCounterSample(sample, nil)
```

Comment:
Is the logics of these methods with non-nil annotation covered somewhere else?

Reply:
I have added tests in places where there is some logic other than just proxying the param. The whole "proxying" logic is tested with an integration test.